src/guess_density.jl

```diff
@@ -32,6 +32,63 @@ guess_density(basis::PlaneWaveBasis) = guess_density(basis, basis.model.atoms)
     from_fourier(basis, ρ / sqrt(model.unit_cell_volume))
 end
 
+
+@doc raw"""
+    guess_spin_density(basis, magnetic_moments)
+
+Magnetic moments should be specified in units of ``μ_B / 2``
+"""
+function guess_spin_density(basis::PlaneWaveBasis, magnetic_moments=[])
+    guess_spin_density(basis, basis.model.atoms, magnetic_moments)
+end
+@timing function guess_spin_density(basis::PlaneWaveBasis{T}, atoms, magnetic_moments) where {T}
+    # TODO Code duplication with guess_density
+    model = basis.model
+    if isempty(magnetic_moments) && model.spin_polarization in (:none, :spinless)
+        return nothing
+    end
+    model.spin_polarization == :collinear ||
+        error("Initial magnetic moments can only be used with collinear models.")
+
+    # If no atoms or magnetic moments, start with a zero spin density
+    ρspin = zeros(complex(T), basis.fft_size)
+
+    # TODO Check how people really do this, I'm not sure this
+    #      is the best possible way ...
```

Comment:
why?

Reply:
Some codes seem to do some integration around atomic positions in a sphere or stuff like that, I'll check that in detail, but not for this PR.